.github/workflows/test.yaml

```diff
@@ -52,8 +52,7 @@ jobs:
           persist-credentials: false
       - name: Install valgrind
         run: sudo apt-get update && sudo apt-get install -y valgrind
-      - run: export VALGRIND="valgrind -q"
-      - run: make test integration
+      - run: VALGRIND=valgrind make test integration
```

Comment:
Good catch :sob: It looks like the `VALGRIND` end manipulation has been inoperable since it was introduced in 456d64bf8f42b76ee21b0f64f895cbf3187b22a5